src/ui-layout.js

```diff
@@ -266,7 +284,7 @@ angular.module('ui.layout', [])
               }
 
               // don't allow the container size to initialize smaller than the minSize
-              if(opts.sizes[i] < opts.minSizes[i]) opts.sizes[i] = opts.minSizes[i];
+              if(!c.collapsed && opts.sizes[i] < opts.minSizes[i]) opts.sizes[i] = opts.minSizes[i];
             }
 
             if(opts.maxSizes[i] != null) {
```

Comment:
same thing agian, would prefer !==
I realize you probably didn't write these LOC.